main.tf

```diff
@@ -87,8 +87,24 @@ resource "aws_lambda_function" "logs_to_datadog" {
       DD_STORE_FAILED_EVENTS = var.store_failed_events
       DD_S3_BUCKET_NAME      = module.datadog_serverless_s3.bucket_name
       ## Filter out lambda platform logs
-      DD_LOGS_CONFIG_PROCESSING_RULES = "[{\"type\": \"exclude_at_match\", \"name\": \"exclude_start_and_end_logs\", \"pattern\": \"(START|END) RequestId\"}]"
-      EXCLUDE_AT_MATCH                = "\"(START|END) RequestId:\\s"
+      DD_LOGS_CONFIG_PROCESSING_RULES = "[{\"type\": \"exclude_at_match\", \"name\": \"exclude_start_and_end_logs\", \"pattern\": \"(START|END) RequestId\" }]"
+      DD_LOGS_CONFIG_PROCESSING_RULES = jsonencode([
```

Comment:
Feel like this is easier to read with multiple rules 🤷

Reply:
We should comment out or remove the `DD_LOGS_CONFIG_PROCESSING_RULES` one above, having it twice won't work.

Reply:
Bah, never went back to delete that after I made the new one 😮‍💨